### ISSUE TITLE
Zoom causes nodes to blur when focus

### ISSUE THREAD

**Issue by alexisrolland** [newcat/baklavajs#249]
Hi there,
I think this issue is related to this other issue which is closed: https://github.com/newcat/baklavajs/issues/178
I am using version `1.9.3` and I somehow still have a blur issue.

I have a simple custom node with one text area option (note this text area is an Ant Design component). It looks good in general, but when zooming in/out and then clicking on the text area to type text (focus), the whole node gets blurred. When clicking outside the text area, then it gets back to normal. This does not happen if I don't zoom in/out.

**Comment by alexisrolland** [comment]
Thank you. Your comment put me on the right track. I did not change the box shadow but removing the `filter` CSS property did the trick for me.

```css
.node {
  filter: none !important; /* This removes the bug where nodes get blurred */
}
```

**Comment by KaKi87** [comment]
I can also reproduce this issue on v2 and workaround it the same way :
```css
.baklava-node {
    filter: none !important;
}
```
But please reopen this issue so that the maintainer fixes it properly at some point.
Thanks

**Comment by newcat** [comment]
Fixed in [v2.0.2-beta.5](https://github.com/newcat/baklavajs/releases/tag/v2.0.2-beta.5)
